backend/test/integration/langchain.test.ts

```diff
@@ -11,23 +22,19 @@ import {
 	promptEvalContextTemplate,
 	promptEvalPrompt,
 } from '@src/promptTemplates';
-import { RetrievalQAChain } from 'langchain/chains';
-import { PromptTemplate } from 'langchain/prompts';
-
-import { ChatOpenAI } from 'langchain/chat_models/openai';
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
-const mockCall = jest.fn();
+const mockCall = jest.fn<any>();
 const mockRetrievalQAChain = {
 	call: mockCall,
 };
 const mockPromptEvalChain = {
 	call: mockCall,
```

Comment:
The problem seems to be that we are needlessly using the same mock function for two different calls. In fact, we are using it for three calls, as we also use it inline in the LLMChain constructor mock. Can we not declare these using inline jest.fn()?

```
const mockPromptEvalChain = {
  call: jest.fn(),
};

expect(mockPromptEvalChain.call).toHaveBeenCalled();
```

Reply:
My latest commit has resolved this :)